examples/cells_quota.ts

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env auraescript
```

Comment:
for christmas (insert festive holiday of your choice) i would like something that executes typescript and checks the cgroup filesystem so we can unit test this stuff :)